### ISSUE TITLE
eventClick return value is ignored

### ISSUE THREAD

**Issue by pragmatick** [angular-ui/ui-calendar#254]
According to http://fullcalendar.io/docs/mouse/eventClick/ returning false in the function should make the browser not follow the link (like preventDefault() or something). This works fine with the original calendar script, but in ui-calendar this is ignored. To reproduce:

Change the calendarDemo.js:
```javascript
$scope.alertOnEventClick = function( date, jsEvent, view){
        $scope.alertMessage = (date.title + ' was clicked ');
		jsEvent.preventDefault();
		return false;
    };```

and click on the "Click for Google". Only the alert should be shown and you should stay on the site, but instead you go to google.

**Comment by LilBiggs** [comment]
I figured out why this is happening, but haven't figured out a solution. I thought maybe if I pointed out the cause someone might know a solution. The issue is caused by the wrapFunctionWithScopeAppy in the uiCalendarCtrl controller.  [Line 34](https://github.com/angular-ui/ui-calendar/blob/master/src/calendar.js#L34). Your alertOnEventClick function is getting wrap in another function. When the outer function calls your funnction via: 
```
$timeout(function(){
     //functionToWrap is really the reference to $scope.alertOnEventClick
     functionToWrap.apply(_this, args); 
});
```
The results of alertOnEventClick is not being return. Therefore your `return false;` statement basically just dies. If this was a non-angular app the fix would be to just remove the $timeout and 'return functionToWrap.apply(_this, args);` But I am sure will would break angular features.

Maybe someone else can chime in with a fix.
